desktop/apps/auctions2/components/auctions_block/AuctionsBlock.styl

```diff
@@ -33,5 +33,36 @@
       avant-garde-size s-headline
       background white
   &__item-body
-    .date
-      avant-garde-size s-headline
+    display flex
+    flex-direction column
+    justify-content space-between
+    height 100%
+  .date
+    avant-garde-size s-headline
+
+@media (max-width: 1250px)
```

Comment:
For responsive styles, this [mixin function](https://github.com/artsy/force/blob/master/desktop/components/stylus_lib/index.styl#L261) was recently added that allows one to do 

```css
.some-class
  background: black

  +respond-mobile() 
    background: green
  
  +respond-desktop()
    background: blue
  
```